RELEASE.rst

```diff
@@ -0,0 +1,8 @@
+RELEASE_TYPE: patch
```

Comment:
I think this needs to be a minor version increment, because it increases the size of the public API (even though it does so by just documenting something that was already there).